.github/workflows/pkgci_regression_test.yml

```diff
@@ -251,61 +245,33 @@ jobs:
             --durations=0 \
             --config-files=${MODELS_CONFIG_FILE_PATH}
 
-      - name: Run external tests - models with real weights and additional flags
-        if: "matrix.models-extra-flags-config-file != '' && !cancelled()"
+      - name: "Running SDXL special model tests"
+        if: "matrix.backend != 'vulkan' && !cancelled()"
```

Comment:
Why check for `!= 'vulkan'`? Maybe check for `== rocm` instead? Or run on all backends?

We can also have pytest marks do filtering for us:
https://github.com/iree-org/iree/blob/6df0372e67b022f021668f3da1c8d9f13495d2f6/.github/workflows/pkgci_regression_test.yml#L215-L217 https://github.com/iree-org/iree/blob/6df0372e67b022f021668f3da1c8d9f13495d2f6/experimental/regression_suite/tests/pregenerated/test_llama2.py#L128-L132
(mark any test case fast enough for presubmit with `presubmit`, choose to also run postsubmit or nightly with other marks)

Reply:
If there are 0 tests selected for a backend, it is failing (rocm or cpu we run)

Reply:
Ah, I see. We could add some tests for Vulkan, but I guess it would only make sense in this test suite if we wanted to set some non-default flags.

I'm thinking ahead to if we also run CUDA and Metal tests - I wouldn't want to expand that to `if: "matrix.backend != 'vulkan' && matrix.backend != 'metal' && matrix.backend != 'cuda' && !cancelled()"`

One idea: create a separate job, decoupled from `test_models`:

* test_onnx
* test_models
* test_regression_suite

or

* test_suites (onnx + models)
* test_regression_models (these "special" models)
* benchmark